packages/backup-restore-operator/charts/templates/deployment.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: apps/v1 
+kind: Deployment
+metadata:
+  name: backup-restore-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    resources.cattle.io/operator: backup-restore
+spec:
+  selector:
+    matchLabels:
+      resources.cattle.io/operator: backup-restore
+  template:
+    metadata:
+      labels:
+        resources.cattle.io/operator: backup-restore
+    spec:
+      serviceAccountName: backup-restore-operator-serviceaccount
+      containers:
+      - name: backup-restore-operator
+        image: {{ template "system_default_registry" . }}{{ .Values.image }}:{{ .Values.tag }}
+        imagePullPolicy: Always
+        env:
+        - name: CHART_NAMESPACE
+          value: {{ .Release.Namespace }}
+          {{- if .Values.pvc }}
+        - name: DEFAULT_PVC_BACKUP_STORAGE_LOCATION
+          value: {{ .Release.Name }}-pvc
+          {{- end }}
+          {{- if .Values.s3 }}
+        - name: DEFAULT_S3_BACKUP_STORAGE_LOCATION
+          value: {{ .Release.Name }}-s3
+          {{- end }}
+        {{- if .Values.pvc }}
+        volumeMounts:
+        - mountPath: "/var/lib/backups"
+          name: pv-storage
+      volumes:
+        - name: pv-storage
+          persistentVolumeClaim:
+            claimName: {{ .Values.pvcName }}
```

Not sure how you have this structured, but this should probably be:
```suggestion
            claimName: {{ .Values.pvc.name }}
```

---

The PVC approach hasn't been finalized, I forgot to mention in the description. QA has started testing my branch for everything else, and they want this to get merged even without PVC changes. So I'll update this with my next PR